uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -104,9 +184,19 @@ sub lookupNextVisitLabel {
     return $visitLabel;
 }
 
-################################################################
-########### getFileNamesfromSeriesUID  #########################
-################################################################
+
+=pod
+
+=head3 getFileNamesfromSeriesUID($seriesuid, @alltarfiles)
+
+?????????????????????????????
+
+INPUT: seriesUID, list of tar files
+
+RETURNS: ???????????????????
```

Comment:
Returns a list of the filenames with a given seriesUID (to be later tarred by the `extract_tarchive` method)